src/server.h

```diff
@@ -1993,13 +1991,15 @@ void restartAOFAfterSYNC();
 /* Child info */
 void openChildInfoPipe(void);
 void closeChildInfoPipe(void);
-void sendChildInfo(int process_type);
+void sendChildInfo(int process_type, int cow_type, size_t cow_size);
```

Comment:
alternative to the "enum" of COW_TYPE_LAST and COW_TYPE_CURRENT, maybe this parameter should just be boolean, "on_exit" true / false?
what do you think?
```suggestion
void sendChildInfo(int process_type, int on_exit, size_t cow_size);
```